<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cameraTabWidget</class>
 <widget class="QWidget" name="cameraTabWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Tab</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="rawCameraVerticalLayout" stretch="0,0,0">
     <item>
      <widget class="QComboBox" name="cameraInputsComboBox">
       <property name="currentText">
        <string/>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calibrateButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Calibrate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="undistortCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Undistort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
